product_order_noname/README.rst

```diff
@@ -0,0 +1,37 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-with-text-154x68.png
```

Comment:
Could you use the README folder new fashion, please?
 - https://github.com/OCA/maintainer-tools/tree/master/template/module/readme